java/maven/src/org/netbeans/modules/maven/api/execute/RunConfig.java

```diff
@@ -79,6 +79,28 @@ public interface RunConfig {
 
     String getActionName();
     
+    /**
+     * Options/switches passed to maven.
+     * @return a read-only copy of the current maven options
+     * @since 2.166
```

Comment:
The current master's version of java/maven is 2.166 as well, this should be increased, 2.166 is over 2 months old already.